tsdb/functions.go

```diff
@@ -23,6 +23,7 @@ import (
 // These are used by the mapFunctions in this file
 type Iterator interface {
 	Next() (time int64, value interface{})
+	Fields() map[string]interface{}
```

Comment:
Am I correct in assuming that the Fields() return value is different after each call to Next?